### DISCUSSION TITLE
Multi-user privacy?

### DISCUSSION THREAD

**Discussion by Arche151** [immich-app/immich#2511] [Q&A]
If users of my hosted Immich server password-protect their gallery, can I, as the admin, still access and view them? 

I intend to set up Immich as a private alternative to Google Photos for my family, and I want to ensure that each family member has true privacy without compromising their images and videos to me.

**Comment by bo0tzz** [comment]
You could access the files on disk, but not inside immich.

**Reply by Arche151** [reply]
Ah ok, too bad.
